apps/zotonic_core/src/zotonic_core.erl

```diff
@@ -127,27 +128,34 @@ mnesia_dir() ->
 
 mnesia_dir_config() ->
     case application:get_env(mnesia, dir) of
-        {ok, none} -> undefined;
-        {ok, ""} -> undefined;
-        {ok, setup} -> mnesia_data_dir();
-        {ok, "data/mnesia"} -> mnesia_data_dir();
-        {ok, "priv/mnesia"} -> mnesia_data_dir();
-        {ok, Dir} -> {ok, Dir};
-        undefined -> mnesia_data_dir()
+        {ok, none} ->
+            undefined;
+        {ok, ""} ->
+            undefined;
+        {ok, setup} ->
+            mnesia_data_dir();
+        {ok, "data/mnesia"} ->
+            mnesia_data_dir();
+        {ok, "priv/mnesia"} ->
+            mnesia_data_dir();
+        {ok, Dir} ->
+            {ok, Dir};
+        undefined ->
+            mnesia_data_dir()
     end.
 
 mnesia_data_dir() ->
-    MnesiaDir = filename:join([ z_config:get(data_dir), atom_to_list(node()), "mnesia" ]),
+    MnesiaDir = filename:join([z_config:get(data_dir), atom_to_list(node()), "mnesia"]),
     case z_filelib:ensure_dir(MnesiaDir) of
         ok ->
             application:set_env(mnesia, dir, MnesiaDir),
             {ok, MnesiaDir};
         {error, Reason} ->
             ?LOG_ERROR(#{
-                text => <<"Could not create mnesia dir">>,
-                path => MnesiaDir,
-                result => error,
-                reason => Reason
-            }),
+                           text => <<"Could not create mnesia dir">>,
+                           path => MnesiaDir,
+                           result => error,
+                           reason => Reason
+                       }),
```

Comment:
Why the extra indentation?  Does it want to push it at the same indent as the `#{` ?

I also like to gave the closing `})` on the same level as the `?LOG_ERROR`, as it is easier to scan upwards to see to what it belongs.

Reply:
That, sadly, is not yet an option for `rebar3_format`. Feel free to open a ticket (or a PR ;)) to add it.